packages/backend/src/build-disk-image.ts

```diff
@@ -304,8 +307,24 @@ export async function getUnusedName(name: string): Promise<string> {
   return unusedName;
 }
 
+export async function getBuilder(): Promise<string> {
+  // use the preference to decide which builder to use
+  const buildProp = await getConfigurationValue<string>('builder');
+
+  if (buildProp === 'RHEL') {
+    return bootcImageBuilderRHEL;
+  }
+
+  // always default to centos bib
+  return bootcImageBuilderCentos;
```

Comment:
should we fail at all if the config setting is not centos or rhel? i see in our tests it still passed.

example you can pass in anything 'foobar' as the configuration setting and it'll always pick centos.

Reply:
I wanted to play it safe, i.e. even if config corrupted => we ignore and use CentOS anyway.